tests/test_on_data/test_recording_interfaces.py

```diff
@@ -539,6 +540,29 @@ class TestTdtRecordingInterface(RecordingExtractorInterfaceTestMixin, TestCase):
     save_directory = OUTPUT_PATH
 
 
+class TestTdtRecordingInterfaceWithGain(RecordingExtractorInterfaceTestMixin, TestCase):
+    data_interface_cls = TdtRecordingInterface
+    interface_kwargs = dict(folder_path=str(DATA_PATH / "tdt" / "aep_05"), gain=1.0)
+    save_directory = OUTPUT_PATH
+
+    def run_custom_checks(self):
+        # Check that the gain is applied
+        recording_extractor = self.interface.recording_extractor
+        gains = recording_extractor.get_channel_gains()
+        expected_channel_gains = [1.0] * recording_extractor.get_num_channels()
```

Comment:
I would use something other than `1.0` to make the test a little clearer, since `1.0` is often a default of other things

Reply:
applied that in: [b95cfaf](https://github.com/catalystneuro/neuroconv/pull/704/commits/b95cfaff1f61ad7d4c9bd2c35ae54b18ff604b95)